sdk/servicebus/azure-servicebus/stress/scripts/stress_test_base.py

```diff
@@ -195,18 +194,18 @@ def _construct_message(self):
             return message
 
     def _send(self, sender, end_time):
-        self._schedule_interval_logger(end_time, "Sender " + str(self))
+        self._schedule_interval_logger(end_time, self.logger, self.process_monitor, "Sender " + str(self))
         try:
-            _logger.debug("Starting send loop")
+            self.logger.debug("Starting send loop")
             # log sender
-            _logger.debug("Sender: {}".format(sender))
+            self.logger.debug("Sender: {}".format(sender))
             with sender:
                 while end_time > datetime.utcnow() and not self._should_stop:
                     try:
                         message = self._construct_message()
                         if self.send_session_id != None:
                             message.session_id = self.send_session_id
-                        _logger.debug("Sending message: {}".format(message))
+                        self.logger.debug("Sending message: {}".format(message))
```

Comment:
for logging we should use the % formatting style to defer formatting till needed https://docs.python.org/3/howto/logging.html#optimization